app/proguard-rules.pro

```diff
@@ -0,0 +1,25 @@
+# Add project specific ProGuard rules here.
```

Comment:
It protects your app by minifying and obfuscating it. Doesn't matter, just don't put on git files you're not familiar with.